x-pack/platform/plugins/shared/fleet/server/types/so_attributes.ts

```diff
@@ -168,6 +168,7 @@ interface OutputSoBaseAttributes {
   output_id?: string;
   ssl?: string | null; // encrypted ssl field
   preset?: OutputPreset;
+  write_to_streams?: boolean;
```

Comment:
nit:   does the naming`write_to_streams`  is explicit enougth? does something like `write_to_logs_streams` will be more explicit? curious to know what naming folks in the streams project will use